consensus/polybft/extra_test.go

```diff
@@ -559,11 +560,11 @@ func TestExtra_CreateValidatorSetDelta_Cases(t *testing.T) {
 func TestExtra_CreateValidatorSetDelta_BlsDiffer(t *testing.T) {
```

Comment:
I extracted all of them except TestExtra_InitGenesisValidatorsDelta because this one would cause issues with circular references. I had to move CreateValidatorSetDelta() into validator package as well.